lib/nerves_runtime/configfs.ex

```diff
@@ -0,0 +1,153 @@
+defmodule Nerves.Runtime.ConfigFS do
+  use GenServer
+  alias Nerves.Runtime
+  require Logger
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  def init([]) do
+    # configfs_home="/sys/kernel/config"
+    # mount none $configfs_home -t configfs
+    case Runtime.cmd("mount", [], :return) do
+      {str, 0} -> String.contains?(str, "none on /sys/kernel/config type configfs")
+      {err, _} -> raise "Failed to check mounted partitions: #{inspect(err)}"
+    end
+    |> unless do
+      Logger.info("Going to mount ConfigFS.")
+      {_, 0} = Runtime.cmd("mount", ["none", "/sys/kernel/config", "-t", "configfs"], :return)
```

Comment:
You can mount special filesystems like this from erlinit.config. Why don't you PR nerves_system_bbb and rpi0 with those mounts.

Reply:
I figured as much. I guess we can assume configfs is mounted by Erlinit.